lib/modules/userHighlight.js

```diff
@@ -256,8 +280,33 @@ module.go = () => {
 	if (module.options.highlightSelf.value && username) {
 		highlight(`author[href$="/${username}"]`, module.options.selfColor.value, module.options.selfColorHover.value);
 	}
+	if (module.options.highlightOpMentions.value && isPageType('comments')) {
+		highlightMentionedUsers();
+	}
 };
 
+function highlightMentionedUsers() {
+	const mentions = getUserLinksFromContainer('#siteTable .usertext-body');
+	mentions.forEach(mention => {
+		const mentionAnchor = ((mention: any): HTMLAnchorElement);
+		const mentionedUsername = getUsernameFromAnchor(mentionAnchor);
+		highlight(`author[href$="/${mentionedUsername}" i]`, module.options.opMentionsColor.value, module.options.opMentionsHover.value);
+	});
+}
+
+function getUserLinksFromContainer(containerSelector: string): NodeList<HTMLElement> {
+	return document.querySelectorAll(`${containerSelector} a[href^="/u/"]`);
```

Comment:
This selector is pretty limited. Could you use the usernameRE here too?

It would probably be easier to refactor these three functions as


1. pull the links from the post text
1. Map links into usernames
    1. attempt to extract the username from the link using the regex
    2. silently return/continue if there's no match.
1. Iterate over usernames to add highlighting (filter no-match / undefined)